### PR TITLE
install nut and miniupnpd;  enable and start services

### PR DESCRIPTION
**Pull request by grahams**

### CODE REVIEW

File: obsdfw/Makefile [hunk 1]
```diff
@@ -92,6 +92,9 @@ obsdfw: \
 /etc/miniupnpd.conf : miniupnpd.conf
 	install -o root -g wheel -m 0644 \
 	  miniupnpd.conf /etc/miniupnpd.conf
+	pkg_add miniupnpd
```
Comment (matthewjweaver) [inline obsdfw/Makefile]: This definitely needs to move above the install, or the default config will eat yours the first time.
I think it's also wise to make this conditional, something like:
`@if ! [ -d /var/db/pkg/miniupnpd* ]; then pkg_add miniupnpd; fi`

File: obsdfw/Makefile [hunk 2]
```diff
@@ -92,6 +92,9 @@ obsdfw: \
...
+	rcctl enable miniupnpd
```
Comment (matthewjweaver) [inline obsdfw/Makefile]: `@if ! rcctl get miniupnpd > /dev/null; then rcctl enable miniupnpd; fi `

File: obsdfw/Makefile (hunk 1 above)
Comment (grahams) [reply obsdfw/Makefile]: lol fyi I was just wayyyy to lazy to figure out A) conditionals in Make and 2) How to determine if it was already installed so tyvm

File: obsdfw/Makefile [hunk 3]
```diff
@@ -140,6 +143,9 @@ obsdfw: \
 /etc/nut/upsmon.conf : upsmon.conf
 	install -o _ups -g wheel -m 0600 \
 	  upsmon.conf /etc/nut/upsmon.conf
+	pkg_add nut
```
Comment (matthewjweaver) [inline obsdfw/Makefile]: as above

File: obsdfw/Makefile [hunk 4]
```diff
@@ -140,6 +143,9 @@ obsdfw: \
...
+	rcctl enable upsmon
```
Comment (matthewjweaver) [inline obsdfw/Makefile]: as above

**Review by matthewjweaver — COMMENTED** [review]
I think some reordering and using some shell conditionals is probably a good idea to save a headache later

whupass otherwise I'll probably rig all mine up this way too

File: obsdfw/Makefile (hunk 1 above)
Comment (matthewjweaver) [reply obsdfw/Makefile]: I think the @ will work to just shell out

**Pull request closed without merging**